mirrord/operator/src/crd/policy.rs

```diff
@@ -78,6 +86,12 @@ pub struct MirrordClusterPolicySpec {
     // TODO: make the k8s list type be set/map to prevent duplicates.
     /// List of features and operations blocked by this policy.
     pub block: Vec<BlockedFeature>,
+
+    /// List of environment variables that should be excluded when using mirrord.
+    ///
+    /// These environment variables won't be retieved from the target even if the user
+    /// specifies them in their `feature.env.include` mirrord config.
+    pub env_vars_exclude: HashSet<String>,
```

Comment:
is it backward compatible? would policies without env_vars_exclude work?

Reply:
Meh, thought it would just have some default value, but it appears in the "required" section.

Changed to `Option`.

Reply:
I think it works if you mark it with `#[serde(default)]`, no?

Reply:
Yup! Removed the `Option`.